src/SlmQueueBeanstalkd/Queue/BeanstalkdQueue.php

```diff
@@ -26,9 +31,13 @@ class BeanstalkdQueue extends AbstractQueue implements BeanstalkdQueueInterface
      * @param string           $name
      * @param JobPluginManager $jobPluginManager
      */
-    public function __construct(Pheanstalk $pheanstalk, $name, JobPluginManager $jobPluginManager)
+    public function __construct(Pheanstalk $pheanstalk, $name, JobPluginManager $jobPluginManager, QueueOptions $options = null)
```

Comment:
Why are the options optional? When using the factory, the options are always injected?